websocket/polygon.go

```diff
@@ -428,12 +428,22 @@ func (c *Client) handleData(eventType string, msg json.RawMessage) {
 		}
 		c.output <- out
 	case "AM":
-		var out models.EquityAgg
-		if err := json.Unmarshal(msg, &out); err != nil {
-			c.log.Errorf("failed to unmarshal message: %v", err)
-			return
+		switch c.market {
+		case Forex, Crypto:
```

Comment:
Interesting that this isn't how it was working prior.

Reply:
Yeah, the `pair` attribute missing when I was QA'ing it tipped me off - they have the same fields otherwise I believe (or at least the same json tags), so you get everything except `pair`.

Reply:
great catch! this part lgtm